android/bridge/src/main/java/tech/berty/gobridge/bledriver/Advertiser.java

```diff
@@ -0,0 +1,187 @@
+package tech.berty.gobridge.bledriver;
```

Comment:
Can you move this package to `android/bridge/src/main/java/ipfs/gomobile/android/bledriver`?